IRSA-README.md

```diff
@@ -0,0 +1,88 @@
+# Domino Advanced Credential Propagation for AWS based on IAM Role for Service Accounts (IRSA)
+
+IAM Role for Service Accounts ([IRSA](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html)) is a capability which allows Kubernetes Service Accounts attached to a EKS workload to assume IAM Role(s). In much the same way as you can attach and IAM Role to an EC2 instance at startup.
+
+## Article Topics
+ - Security 
+ - Identity And Access Management (IAM) 
+ - Role Based Access Control (RBAC)
+ - OpenID Connect (OIDC), Web Identity and Attribute Based Access Control (ABAC) 
```

Comment:
Not quite, the kubernetes SA token that is sent to AWS is an Auth token, not an access token, which is what makes it OAuth instead of OIDC. The terminology is confusing i agree :-) 
https://supertokens.com/blog/oauth-vs-oidc
`While OAuth describes what resources a user has access to, OIDC describes who the user is`
Basically any token that allows you to access a resource is an OAuth token. structurally they're both just JWTs though

Reply:
Got it. So should I reword this as-
`OAuth, Web Identity and Attribute Based Access Control (ABAC) `